wled00/button.cpp

```diff
@@ -7,11 +7,13 @@
 #define WLED_DEBOUNCE_THRESHOLD      50 // only consider button input of at least 50ms as valid (debouncing)
 #define WLED_LONG_PRESS             600 // long press if button is released after held for at least 600ms
 #define WLED_DOUBLE_PRESS           350 // double press if another press within 350ms after a short press
-#define WLED_LONG_REPEATED_ACTION   300 // how often a repeated action (e.g. dimming) is fired on long press on button IDs >0
+#define WLED_LONG_REPEATED_ACTION   400 // how often a repeated action (e.g. dimming) is fired on long press on button IDs >0
 #define WLED_LONG_AP               5000 // how long button 0 needs to be held to activate WLED-AP
 #define WLED_LONG_FACTORY_RESET   10000 // how long button 0 needs to be held to trigger a factory reset
+#define WLED_LONG_BRI_STEPS          16 // how long to wait before increasing/decreasing brightness on long press
```

Comment:
That is a large step. Perhaps take a look at `ir.cpp` how it is implemented there.
IMO, if you keep it compile time dependant, it would be better if this would be lower (i.e. 8) to allow more gradual brightness.

I also find comment misleading. It has nothing to do with "time".

Reply:
I chose to double the step since I found the previous stepping to be too slow, and this with the new timing of the action was a good trade off for me, 16 different levels of brightness are more than enough (IMHO) when controlling through a physical button, as a user I'd rather go from 1 to max quicker. 
But obviously it can be reverted to 8!

I agree, the comment is wrong and misleading, I must have turned my brain off when writing it. Will fix it asap